### PR TITLE
[L09] Misleading comments and typographical errors

### PR DESCRIPTION
**Pull request by yorhodes**
### Description

_A few sentences describing the overall effects and goals of the pull request's commits.
What is the current behavior, and what is the updated/expected behavior with this PR?_

### Other changes

_Describe any minor or "drive-by" changes here._

### Tested

_An explanation of how the changes were tested or an explanation as to why they don't need to be._

### Related issues

- Fixes https://github.com/celo-org/celo-labs/issues/605

### Backwards compatibility

_Brief explanation of why these changes are/are not backwards compatible._

### CODE REVIEW

File: packages/protocol/contracts/governance/Validators.sol [hunk 1]
```diff
@@ -918,7 +918,7 @@ contract Validators is
     uint256 balance = getLockedGold().getAccountTotalLockedGold(account);
     // Add a bit of "wiggle room" to accommodate the fact that vote activation can result in a 1
     // wei rounding error.
-    return balance.add(10) >= getAccountLockedGoldRequirement(account);
+    return balance.add(1) >= getAccountLockedGoldRequirement(account);
```
Comment (yorhodes) [inline packages/protocol/contracts/governance/Validators.sol]: @asaj any rationale for 10 as opposed to 1?
does it have to do with multiple vote activations?
this feels like an edge case that we don't account for elsewhere

File: packages/protocol/contracts/common/TransferWhitelist.sol [hunk 2]
```diff
@@ -34,7 +34,7 @@ contract TransferWhitelist is Ownable, UsingRegistry {
 
   /**
    * @notice Remove an address from the whitelist.
-   * @param removedAddress The address to add.
```
Comment (aaronmgdr) [inline packages/protocol/contracts/common/TransferWhitelist.sol:37]: ‼️

File: packages/protocol/contracts/governance/Validators.sol (hunk 1 above)
Comment (m-chrzan) [reply packages/protocol/contracts/governance/Validators.sol]: My guess would be to for sure overshoot whatever the rounding error can be? But yeah, would love @asaj's input before this gets merged.

Comment (asaj) [reply packages/protocol/contracts/governance/Validators.sol]: Yeah it was just for an additional margin of safety. I would argue for keeping it given I don't see much of a benefit of reducing it to 1, but I don't feel terribly strongly

**Pull request merged**